app/code/Magento/Directory/Model/Currency/Import/FixerIo.php

```diff
@@ -23,8 +23,13 @@ class FixerIo extends AbstractImport
     /**
      * @var string
      */
-    public const CURRENCY_CONVERTER_URL = 'http://data.fixer.io/api/latest?access_key={{ACCESS_KEY}}'
-        . '&base={{CURRENCY_FROM}}&symbols={{CURRENCY_TO}}';
+    public const CURRENCY_CONVERTER_URL = 'https://api.apilayer.com/fixer/latest?symbols{{CURRENCY_TO}}' .
+                                          '&base={{CURRENCY_FROM}}';
+    
+    /**
+     * @var string
+     */
+    public const API_KEY_CONFIG_PATH = 'currency/fixerio/api_key';
```

Comment:
Let's make it private due to backward compatibility reasons
```suggestion
    private const API_KEY_CONFIG_PATH = 'currency/fixerio/api_key';
```